meinberlin/apps/documents/templates/meinberlin_documents/chapter_detail.html

```diff
@@ -2,93 +2,58 @@
-                                <a
-                                        class="btn btn--primary"
-                                        href="{{ paragraph.get_absolute_url }}"
-                                >
-                                    <i class="far fa-comment" aria-hidden="true"></i>
-                                    {% blocktranslate count count=paragraph.comments.all|length %}{{ count }} Comment{% plural %}{{ count }} Comments{% endblocktranslate %}
-                                </a>
+    <article class="u-spacer-top">
+        {% if chapter_list %}    
+            <div class="modul-toc">
+                <h2 class="title">{% translate 'Table of Contents' %}</h2>
+                <ol aria-label="{% translate 'Table of Contents' %}">
+                    {% for chapter in chapter_list %}
+                    <li>
+                        <a href="{{ chapter.get_absolute_url }}">
+                            {{ chapter.name }}
+                        </a>
+                    </li>
+                {% endfor %}
+                </ol>
+            </div>
+            <div>
+                <h2>{{ object.name }}</h2>
+                {% for paragraph in object.paragraphs.all %}
+                    <section id="paragraph-{{ object.pk }}-{{ paragraph.pk }}">
+                        <div>
+                            <h3>
+                                {{ paragraph.name }}
+                            </h3>
+                            <div class="ck-content">
+                                {{ paragraph.text|richtext }}
                             </div>
-                        </section>
-
-                    {% endfor %}
-                </div>
+                        </div>
+                        <div class="u-flex-end">
+                            <a href="{{ paragraph.get_absolute_url }}">
+                                <i class="far fa-comment" aria-hidden="true"></i>
+                                {% blocktranslate count count=paragraph.comments.all|length %}{{ count }}
+                                Comment{% plural %}{{ count }}
+                                Comments{% endblocktranslate %}
+                            </a>
+                        </div>
+                    </section>
+                {% endfor %}
             </div>
-        </div>
         {% else %}
-        <div class="container">
-            <div class="offset-lg-2 col-lg-8">
-                {% translate 'A Document has not been added yet' %}
-            </div>
-        </div>
+            {% translate 'A Document has not been added yet' %}
         {% endif %}
     </article>
 
     {% if chapter_list|length > 1 %}
-        <div class="control-bar__bottom-overlap">
-            {% include 'meinberlin_documents/includes/nextprev.html' %}
-        </div>
+        {% include 'meinberlin_documents/includes/nextprev.html' %}
     {% endif %}
 
-
     <div class="container">
```

Comment:
`container` and `offset-lg-2 col-lg-8` are bootstrap classes and shouldn't be here anymore, right?

Reply:
@goapunk I just talked to @hom3mad3 on Mattermost :)